actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -95,8 +95,8 @@ def distance_of_time_in_words(from_time, to_time = 0, options = {})
           scope: :'datetime.distance_in_words'
         }.merge!(options)
```

Comment:
Is there a reason to mutate the new hash rather than doing `options.merge(scope: :'datetime.distance_in_words')`? I don't particularly care, just curious.